db/migrate/20170817095425_add_temporary_timestamps_to_edition.rb

```diff
@@ -0,0 +1,15 @@
+class AddTemporaryTimestampsToEdition < ActiveRecord::Migration[5.1]
+  def up
```

Comment:
I think actually you only need a `change` method which can be the same as the `up` method here, and the `down` method can be determined automagically.